tests/unittests/OperatorTest.cpp

```diff
@@ -3362,6 +3362,65 @@ TEST_P(InterpAndCPU, insertTensorTest) {
   }
 }
 
+/// Test RowwiseQuantizedFullyConnected Node.
+TEST_P(InterpOnly, rowwiseQuantizedFCTest) {
+  // In this test we subtract the outputs of a row-wise quantized FC and a
+  // floating-point FC and ensure that the error is below some low value.
+  Context ctx;
+  auto *input =
+      mod_.createPlaceholder(ElemKind::FloatTy, {1, 300}, "in", false);
+  auto *fc = F_->createFullyConnected(ctx, "FC", input, 30);
+  auto *res = mod_.createPlaceholder(ElemKind::FloatTy, fc->getResult().dims(),
+                                     "res", false);
+
+  auto *weights = llvm::cast<Placeholder>(fc->getWeights());
+  auto *bias = llvm::cast<Placeholder>(fc->getBias());
+
+  auto *inputTensor = ctx.allocate(input);
+  auto *weightsTensor = ctx.get(weights);
+  auto *biasTensor = ctx.get(bias);
+  auto *resTensor = ctx.allocate(res);
+
+  inputTensor->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  biasTensor->getHandle().randomize(0, 0.00001, mod_.getPRNG());
```

Comment:
Why bias is so small?

So we have a code, doing 3 things: X, Y and Z. Ideally, you want to have a test, such that if you remove X-functionality from your code (so that it only does Y and Z), than test fails. Same for Y and Z. In other words, test must fail if code "forgets" to do X or Y or Z.

Right now if we remove "bias computation" (`sum += B`) code from Interpreter, the test will still pass. This is not good, because it means that we don't cover it. I have a feeling that if you increase bias magnitude, then we'll have coverage.